app/src/main/res/layout/item_page_list_entry.xml

```diff
@@ -109,6 +83,36 @@
 
     </FrameLayout>
 
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/page_list_item_selected_image"
+        android:layout_width="@dimen/view_list_card_item_image"
+        android:layout_height="@dimen/view_list_card_item_image"
+        android:layout_gravity="center"
+        android:layout_marginEnd="@dimen/list_item_horizontal_padding"
+        android:background="@drawable/button_shape_gray"
+        android:contentDescription="@null"
+        android:padding="8dp"
+        android:tint="@android:color/white"
+        android:visibility="gone"
+        app:backgroundTint="@color/accent50"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="@id/page_list_item_title"
+        app:srcCompat="@drawable/ic_check_black_24dp"
+        tools:visibility="visible" />
+
+    <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/page_list_item_image"
+        style="@style/SimpleDraweeViewPlaceholder"
+        android:layout_width="@dimen/view_list_card_item_image"
+        android:layout_height="@dimen/view_list_card_item_image"
+        android:layout_marginEnd="@dimen/list_item_horizontal_padding"
+        app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
I think the `app:layout_constraintBottom_toBottomOf="parent"` can be removed so that the "download" icon and the image will have the correct alignment to the top.

Reply:
It happens when the description has more than one line.
![Screenshot_1559773405](https://user-images.githubusercontent.com/2435576/58994731-62989b00-87a6-11e9-8dba-416b6b96af06.png)

Reply:
if I remove any constraints it will collapse to the right top corner when 'gone'

Reply:
Actually that is resolved too after I changed the invisible/gone logic. So removed.Done

Reply:
> if I remove any constraints it will collapse to the right top corner when 'gone'
Could you please provide some example?